routers/private/hook_post_receive.go

```diff
@@ -193,6 +186,14 @@ func HookPostReceive(ctx *gitea_context.PrivateContext) {
 					}
 					baseRepo = repo.BaseRepo
 				}
+
+				if !baseRepo.AllowsPulls() {
```

Comment:
I think this logic is still not completely correct, but better than now:
Assume we have repo A and its fork B.
A disables PRs, B allows them.
With the current approach, no message will be shown as A does not allow creating PRs.
However, it might still be intended that a PR on B is opened.

Reply:
Indeed, that should be fixed as well.